include/nlohmann/detail/input/binary_reader.hpp

```diff
@@ -2514,7 +2514,7 @@ class binary_reader
     /// the number of characters read
     std::size_t chars_read = 0;
 
-    /// whether we can assume little endianess
+    /// whether we can assume little endianness
```

Comment:
Now this of course is a controversial change, given there is a helper named `little_endianess`. It seems the [more common spelling is with two n-s](https://en.wikipedia.org/wiki/Endianness).

Reply:
I agree: it should be `endianness`. Feel free to also rename the helper.

Reply:
Do you think any user of the library could depend on this? If so this may be a breaking change.

Reply:
No, that function is in the `detail` namespace that nobody should depend on.